addons/website_event/views/event_svg.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="website_event.event_svg">
+        <svg width="211" height="179" viewBox="0 0 211 179" fill="none" xmlns="http://www.w3.org/2000/svg">
+            <g id="Frame" clip-path="url(#clip0_56_106)">
+            <g fill="var(--o-color-1)" opacity="1">
```

Comment:
Provide a fallback for all these `fill` attributes so we can debug outside of the website environment, eg:

```suggestion
            <g fill="var(--o-color-1, #000)" opacity="1">
```